Initial transition to empty screen

apustovitovsky opened the pull request:
Добавил переход на модуль Profile при запуске приложения.

apustovitovsky committed:
Initial transition to empty screen

Sorix commented on TravelTracker/Modules/ProfileModule/ProfileModuleFactory.swift:
```diff
@@ -13,7 +13,7 @@ final class ProfileModuleFactory: Factory {
     func build(with context: Any?) throws -> ProfileViewController {
```
Если не получается убрать вообще контекст, то обычно убирают аргумент.

```suggestion
    func build(with _: Any?) throws -> ProfileViewController {
```

apustovitovsky replied:
Исправлено в 56861ec33baa9fae6815c2dcd9b1080f3ae6c03c

Sorix commented on TravelTracker/Modules/ProfileModule/ProfileViewController.swift:
```diff
@@ -13,9 +13,9 @@ protocol ProfileViewControllerProtocol: AnyObject {
     
     private let presenter: PresenterProtocol
-    private let customView: PasscodeViewProtocol
+    private let customView: UIView
```
Лучше view инициализировать через `private lazy var customView = UIView()`, чтобы не инициализировать их при создании UIViewController.

apustovitovsky replied:
Исправлено в 3ac6f84c0d8750249b1760fa79959119cf1aecfa

Sorix commented on TravelTracker/Modules/ProfileModule/ProfileViewController.swift:
```diff
@@ -41,7 +41,7 @@ final class ProfileViewController: UIViewController {
     
     func configure(with model: PasscodeModel) {
-        customView.configure(with: model)
+        //customView.configure(with: model)
```
👎🏻

apustovitovsky replied:
Исправлено в 3ac6f84c0d8750249b1760fa79959119cf1aecfa

apustovitovsky committed:
Аргумент метода build для ProfileModuleFactory сделан игнорируемым

apustovitovsky committed:
Изменен способ нициализации UIView

Вместо конструктора создаем как lazy свойство. Также убран комментарий и лишние переносы строк/
